examples/rpi-pico-w/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+#![no_std]
```

Comment:
i'd just copy-paste this into each example, even if that means there's a bit of duplication. "single crate with both lib and bin targets" can be confusing for Rust newcomers, it's nice that examples are as simple as possible as a self-contained `.rs` file.

Reply:
true